exampleCourse/questions/fibonacciEditorWithR/tests/tests.R

```diff
@@ -0,0 +1,47 @@
+# Unit tests with `testthat` use the behavior-driven development (BDD) format
+# instead of the traditional `test_that` format. This allows for two "contexts"
+# to be created.
+
+# In particular, we make the following assumptions about unit tests:
+
+# 1. `describe()` function indicates a single problem steps worth.
+# 2. `it()` function describes the specification being assessed.
+#
+# Avoid using multiple comparisons within the same `describe()`
+
+# Sample test
+describe("1", {                               # Specify points per check
+    it("Does `fib()` exist?", {               # Description of test
```

Comment:
I don’t think we support markdown here (the backticks around fib()).

Reply:
This was just styling to pop the function name.

Reply:
Looking at `elements/pl-external-grader-results/pl-external-grader-results.mustache`, we include the description with `{{description}}`, so we’ll escape any HTML before it reaches the student. I’m not sure whether we want styling to make it through? @nwalters512?